### ISSUE TITLE
parallelize message send to large group

### ISSUE THREAD

**Issue by darrenwee** [darrenwee/andreabot#5]
There should be a better way to broadcast messages to a group. With the current list of ~20 recipients, a `/yell message` command takes around 40-60 seconds to fully propagate as each message is sent only after the other.

Might investigate asynchronous functionality from `PyTelegramBotAPI` to speed this up as the broadcasts are not dependent on each other. Low priority but nice to have. Might need to upsize the DigitalOcean droplet to get more CPU threads.

**Comment by darrenwee** [comment]
Fixed this by shifting from `pyTelegramBotAPI` to `telepot` which is much, much faster. A yell propagation takes a few seconds now.
